### PR TITLE
Add npm run clean script

### PR DESCRIPTION
**Pull request by dharit-tan**
### Description
I find myself wanting to blast all `node_modules` and build folders frequently, this would make it easier.

### Dragons

* is it sketch to have `rm -rf` in a script?
* Could not figure out where the build folder for `web` lives.
* This won't do everything that an `npm run clean` in `mobile` will but that's ok.

### How Has This Been Tested?

Ran locally followed by `npm i`.

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by dylanjeffers** [comment]
maybe another option is to add a "clean" method to each sub package so it's not all or nothing?

**Review by raymondjacobson — APPROVED** [review]
web is

build, build-staging, and build-production

**Comment by dharit-tan** [comment]
> maybe another option is to add a "clean" method to each sub package so it's not all or nothing?

I guess I run into it more with mobile, bc I want to blast mobile, common, and then also have to blast web sometimes due to web common.

**Comment by rickyrombo** [comment]
I also echo doing `rm -rf node_modules & rm -rf packages/*/node_modules` rather than any `cd`s

**Comment by dharit-tan** [comment]
> Personally I would expect `clean` as only meaning build artifacts, not node_modules, so maybe better to have separate `npm run clean` and `npm run clean:modules` or something? The former should probably call upon the mobile `npm run clean` script rather than manually doing `rm -rf`s imo? can do `react-native clean-project-auto` to avoid interactive element: https://www.npmjs.com/package/react-native-clean-project

Good call - the `auto` doesn't include the brew update so that'll work. I like this idea.

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "install-hooks": "mookme init --only-hook --skip-types-selection",
     "lint": "lerna run lint",
     "lint:fix": "lerna run lint:fix",
+    "clean": "rm -rf node_modules && cd packages/web && rm -rf node_modules && cd ../mobile && rm -rf node_modules && rm -rf android/build && rm -rf ios/build && cd ../common && rm -rf node_modules && cd ../stems && rm -rf node_modules && cd ..",
```
Comment (sddioulde) [inline package.json:12]: Should the end be "cd ../.."

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Review by rickyrombo — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (rickyrombo) [reply package.json]: yeah probably unnecessary? tbh not sure whether npm commands change your actual terminal's working dir

**Review by dylanjeffers — APPROVED** [review]
dope!

**Pull request merged**